marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,4 @@
+Header header
+
+bool   healthy
+string message
```

Comment:
Anybody think it might be a good idea to add an enum to indicate a status level, such as OK / WARN / ERROR / UNKNOWN?  Or is that getting a little bit too detailed for what this is supposed to be?